src/main/java/space/testflight/FlywayExtension.java

```diff
@@ -127,6 +127,8 @@ public void afterAll(ExtensionContext context) throws Exception {
     if (getDatabaseInstance(context) == Flyway.DatabaseInstanceScope.PER_TEST_CLASS) {
       teardownDb(context, false);
     }
+
+    getGlobalStore(context).get(STORE_CONTAINER, AutoCloseable.class).close();
```

Comment:
What does happen, if the container is already stopped?

Reply:
Testcontainers checks if the container is running, before it tries to close and remove it.
Running this line multiple times, works also flawlessly.

See https://github.com/testcontainers/testcontainers-java/blob/testcontainers-1.1.5/core/src/main/java/org/testcontainers/utility/ResourceReaper.java#L92